components/LoanForm/LoanFormBetterTerms/LoanFormBetterTerms.tsx

```diff
@@ -19,10 +19,10 @@ import { Explainer } from './Explainer';
 import { ethers } from 'ethers';
 import { daysToSecondsBigNum, secondsBigNumToDays } from 'lib/duration';
 import { yupResolver } from '@hookform/resolvers/yup';
-import { Balance } from '../Balance';
 import { formattedAnnualRate } from 'lib/interest';
 import { LoanTermsDisclosure } from 'components/LoanTermsDisclosure';
 import { useLoanDetails } from 'hooks/useLoanDetails';
+import { truncate } from 'lodash';
```

Comment:
nit, this style of import causes all of lodash to get imported, should do `import truncate from 'lodash/truncate'` or similar